pyproject.toml

```diff
@@ -54,6 +54,7 @@ python-multipart = {version = "^0.0.5", optional = true}
 sanic = {version = ">=20.12.2", optional = true}
 aiohttp = {version = "^3.7.4.post0", optional = true}
 fastapi = {version = ">=0.65.2", optional = true}
+starlite = {version = ">=1.44.0", optional = true, python = "^3.8"}
```

Comment:
Should update these constraints everywhere to `>=1.45.1`

Reply:
@Goldziher Should I update to 1.48.0 now?